README.md

```diff
@@ -112,6 +112,20 @@ The supplier frontend runs on port 5003. Use the app at [http://127.0.0.1:5003/]
 To use feature flags, check out the documentation in (the README of)
 [digitalmarketplace-utils](https://github.com/alphagov/digitalmarketplace-utils#using-featureflags).
 
+### Configuring boto
+
+[boto](https://github.com/boto/boto) provides a Python interface to Amazon Web Services; it's what we're using to download from and upload to our s3 buckets.
+
+If don't [configure your AWS credentials correctly](http://boto.readthedocs.org/en/latest/boto_config_tut.html?highlight=~/.aws/credentials#credentials)
+on your local machine, you'll probably run into a nasty-looking `boto.exception.NoAuthHandlerFound` page at some point.
+
+The short version is that you should create an `~/.aws/credentials` file formatted like so:
+```bash
+[default]
+aws_access_key_id = ...
```

Comment:
Which keys should we use? Maybe specify to have the dev account keys rather than prod?  Or is that obvious?